src/main/index.ts

```diff
@@ -3,6 +3,8 @@ import { configStore } from '../shared/configureStore';
 declare const MAIN_WINDOW_WEBPACK_ENTRY: any;
 declare const MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY: any;
 
+require('dotenv').config();
```

Comment:
Is this still needed?

Reply:
yep, renderer and main don't share them. previously env variables were passed down from renderer in functions calling API which just makes them have more params than necessary